src/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,602 @@
+// Navigation ------------- //
+
+$height-nav-secondary: 5rem;
+$width-nav-sidebar:    250px;
+
+
+.usa-header {
+  width: 100%;
+  z-index: 1;
```

Comment:
Some ideas on how to create variable names:
- Based on the actual components: https://css-tricks.com/handling-z-index/
- Based on where in the stacking order; background, foreground, above ground, etc.